common/src/main/java/com/wynntils/mc/mixin/GuiMixin.java

```diff
@@ -167,4 +170,33 @@ private void onRenderHeartsPre(
             ci.cancel();
         }
     }
+
+    @Inject(
+            method = "Lnet/minecraft/client/gui/Gui;renderSelectedItemName(Lnet/minecraft/client/gui/GuiGraphics;)V",
```

Comment:
```suggestion
            method = "renderSelectedItemName(Lnet/minecraft/client/gui/GuiGraphics;)V",
```

Generally, we do this weird, semi-qualified method naming for mixins. I can see the benefits. If you only used the method names, things can break and go unnoticed when porting. The fully qualified name's first part, the holder class itself, however, has no meaningful use here.

Reply:
Oops, Minecraft dev plugin copies the whole thing, didn't think to check how we usually do it :)